include/dxc/dxctools.h

```diff
@@ -63,21 +61,19 @@ CLSID_SCOPE const CLSID
                          0x40b3,
                          {0x96, 0x8d, 0x93, 0xe1, 0x24, 0x73, 0x4d, 0xa4}};
 
-struct __declspec(uuid("261afca1-0609-4ec6-a77f-d98c7035194e"))
-IDxcRewriter2 : public IDxcRewriter {
+INTERFACE_STRUCT_HEADER(IDxcRewriter2, 261afca1,0609,4ec6, a7,7f, d9,8c,70,35,19,4e)
```

Comment:
For any reviewer considering verifying that the new sequences match the old, I have already done so and convinced myself they are correct. Feel free to review them anyway if you don't trust me. 😆

Reply:
Be aware I have yet to fix the formatting (most notably mixed spacing) on these though. Unfortunately I can't format the entire file, leading to manually selecting every header and pressing the autoformat hotkey... :no_mouth: